### DISCUSSION TITLE
Add support for Binance Wallet

### DISCUSSION THREAD

**Discussion by mrq2000** [rainbow-me/rainbowkit#745] [Q&A]
It's very popular crypto wallet extension and i want to integrate it with rainbow on my website.

Did anyone used to custom binance wallet with rainbow or got any idea, please help me.

Thank you,

**Comment by SimonVillage** [comment]
Looking for the same, is it possible to write a custom solution for this easily?

Edit: Seems like there is already support for the chain in the new wagmi version: https://github.com/wagmi-dev/wagmi/discussions/914#discussioncomment-4336942
